models/macros.yaml

```diff
@@ -2,15 +2,62 @@ macros:
   - name: array_agg
     inputs:
       - column_name
-    value: "array_agg( distinct {{column_name}})"
+    value: "
+      {% if warehouse.DatabaseType() == \"redshift\" %}
+        listagg( distinct {{column_name}})
+      {% elif warehouse.DatabaseType() == \"bigquery\" %}
+        array_agg( distinct {{column_name}} IGNORE NULLS)
```

Comment:
Why are we not doing `IGNORE NULLS` in all the cases ?

Reply:
rest of the warehouses by default ignores null values in the output.